src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -74,6 +78,42 @@ void ov::npuw::s11n::write(std::ostream& stream, const ov::Output<const ov::Node
     write(stream, var.get_names());
 }
 
+enum class AnyType : int { STRING = 0, CHARS, INT, UINT32, INT64, UINT64, SIZET, FLOAT, BOOL };
+
+void ov::npuw::s11n::write_any(std::ostream& stream, const ov::Any& var) {
+    // FIXME: figure out a proper way to serialize Any (for config)
+    if (var.is<std::string>()) {
+        write(stream, int(AnyType::STRING));
+        write(stream, var.as<std::string>());
+    } else if (var.is<const char*>()) {
+        write(stream, int(AnyType::CHARS));
+        write(stream, std::string(var.as<const char*>()));
+    } else if (var.is<std::size_t>()) {
+        write(stream, int(AnyType::SIZET));
+        write(stream, var.as<std::size_t>());
+    } else if (var.is<int>()) {
+        write(stream, int(AnyType::INT));
+        write(stream, var.as<int>());
```

Comment:
this may be not portable, but let's keep it aside as we don't claim any blob portability at the moment.